game-core/src/main/java/games/strategy/triplea/attachments/TerritoryEffectAttachment.java

```diff
@@ -98,6 +101,36 @@ public int getCombatEffect(final UnitType type, final boolean defending) {
     return defending ? combatDefenseEffect.getInt(type) : combatOffenseEffect.getInt(type);
   }
 
+  private void setMovementCostModifier(final String value) throws GameParseException {
+    final String[] s = splitOnColon(value);
+    if (s.length < 2) {
+      throw new GameParseException(
+          "movementCostModifier must have a count and at least one unitType" + thisErrorMsg());
```

Comment:
An example of a valid syntax would perhaps make this exception message far more useful and easier to resolve if it were to happen. Otherwise I notice that "must have a count" might not be clear.

Reply:
Count is a pretty common term in the game XML. This message aligns with the rest of the messages in this class so I think its better to leave it.